bin/node/runtime/src/lib.rs

```diff
@@ -584,8 +584,8 @@ pallet_staking_reward_curve::build! {
 
 parameter_types! {
 	pub const SessionsPerEra: sp_staking::SessionIndex = 6;
-	pub const BondingDuration: sp_staking::EraIndex = 24 * 28;
-	pub const SlashDeferDuration: sp_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
+	pub const BondingDuration: sp_staking::EraIndex = 28 * 24 / 6; // 28 days; Era is 6h long
+	pub const SlashDeferDuration: sp_staking::EraIndex = BondingDuration::get() / 4;
```

Comment:
This needs to be smaller than BondingDuration and it's enforced by Staking pallet (tests fail otherwise).
We don't really do any Slashing, so exact value doesn't really matter.